crates/ruff_benchmark/benches/red_knot.rs

```diff
@@ -284,8 +284,8 @@ fn benchmark_many_string_assignments(criterion: &mut Criterion) {
                             s += "attr9"
                         if x.attr10:
                             s += "attr10"
-                        if x.attr11:
-                            s += "attr11"
+                        if len(s) > 0:
+                            s = s[:-3]
```

Comment:
Should we maybe add parts of your PR description as a (Python or Rust) code comment here? If I came across this benchmark without the explanation, I might have missed that the `len` call is (now) a crucial part of that benchmark.